demo/app/examples/multi.component.ts

```diff
@@ -76,6 +76,25 @@ import { DataService } from '../shared/data.service';
             </ng-template>
         </ng-select>
         ---
+        <hr/>
+
+        <label>Custom template for all selected items</label>
+        ---html,true
+        <ng-select
+            #api
+            [items]="githubUsers$ | async"
+            [multiple]="true"
+            bindLabel="login"
+            placeholder="Select items"
+            [(ngModel)]="selectedUsers">
+            <ng-template ng-multi-label-tmp let-item="item" let-items="items">
```

Comment:
remove `let-item="item"`

Reply:
<img width="508" alt="screen shot 2018-03-14 at 10 33 47" src="https://user-images.githubusercontent.com/17824796/37391233-3bc29696-2773-11e8-9c98-54fac02dad19.png">
make demo of something like this?

Reply:
```html
<ng-template ng-multi-label-tmp let-items="items">
    <div class="ng-value" *ngFor="let item of (items ? items.slice(0,4): [])">
        <span class="ng-value-label"><img [src]="item.value.avatar_url" width="20px" height="20px"> {{item.value.login}}</span>
        <span class="ng-value-icon right" (click)="clear(item.value)" aria-hidden="true">×</span>
    </div>
    <div class="ng-value" *ngIf="items.length > 4" >
        <span class="ng-value-label">{{items.length - 4}} more...</span>
    </div>
</ng-template>
```

Reply:
(click)="clear(item.value)" not working. whats the wrong in it.

Reply:
```suggestion
            <ng-template ng-multi-label-tmp let-items="items" let-clear="clear">
```